addons/sale_stock/tests/test_anglo_saxon_valuation.py

```diff
@@ -1392,6 +1392,97 @@ def test_fifo_return_and_credit_note(self):
         self.assertEqual(cogs_aml.debit, 0)
         self.assertEqual(cogs_aml.credit, 20, 'Should be to the value of the returned product')
 
+    def test_fifo_return_and_create_invoice(self):
+        """
+        When creating an invoice for a returned product, the value of the anglo-saxo lines
+        should be based on the returned product's value
+        """
+        self.product.categ_id.property_cost_method = 'fifo'
+
```

Comment:
```suggestion
self.product.invoice_policy = 'delivery'

```
Doing so, if you run the test without your fix, it will fail